doc/modules/partial_dependence.rst

```diff
@@ -163,16 +170,87 @@ support it, and 'brute' is used for the rest.
     samples differently. Remember, however, that the primary assumption for
     interpreting PDPs is that the features should be independent.
 
-.. rubric:: Footnotes
 
-.. [1] For classification, the target response may be the probability of a
-   class (the positive class for binary classification), or the decision
-   function.
+Individual conditional expectation (ICE) plot
+=============================================
+
+Similar to a PDP, an individual conditional expectation (ICE) plot
+shows the dependence between the target function and a target feature.
+However, unlike a PDP, which shows the average effect of the target
+feature, an ICE plot visualizes the dependence of the prediction on a
+feature for each instance separately with one line per instance.
+Due to the limits of human perception, only one target feature is
+supported for ICE plots.
+
+The figures below show four ICE plots for the California housing dataset,
+with a :class:`GradientBoostingRegressor
+<sklearn.ensemble.GradientBoostingRegressor>`. The second figure plots
+the corresponding PD line overlaid on ICE lines.
+
+.. figure:: ../auto_examples/inspection/images/sphx_glr_plot_partial_dependence_004.png
+   :target: ../auto_examples/inspection/plot_partial_dependence.html
+   :align: center
+   :scale: 70
+
+.. figure:: ../auto_examples/inspection/images/sphx_glr_plot_partial_dependence_005.png
+   :target: ../auto_examples/inspection/plot_partial_dependence.html
+   :align: center
+   :scale: 70
+
+While the PDPs are good at showing the average effect of the target features,
+they can obscure a heterogeneous relationship created by interactions.
+When interactions are present the ICE plot will provide much more insight.
+For example, we could observe a linear relationship between the median income
+and the house price in the PD line. However, the ICE lines show that there
+are some exceptions, where the house price remains constant in some ranges of
+the median income.
+
+Similar to PDPs, the target features are assumed to be independent from the
+complement features.
+
+The :mod:`sklearn.inspection` module's :func:`plot_partial_dependence`
+convenience function can be used to create ICE plots by setting
+``individual=True``. In the below example we show how to create a grid of
+ICE plots:
+
+    >>> from sklearn.datasets import make_hastie_10_2
+    >>> from sklearn.ensemble import GradientBoostingClassifier
+    >>> from sklearn.inspection import plot_partial_dependence
+
+    >>> X, y = make_hastie_10_2(random_state=0)
+    >>> clf = GradientBoostingClassifier(n_estimators=100, learning_rate=1.0,
+    ...     max_depth=1, random_state=0).fit(X, y)
+    >>> features = [0, 1]
+    >>> plot_partial_dependence(clf, X, features,
+    ...     individual=True) #doctest: +SKIP
```

Comment:
```suggestion
    ...     individual=True)  # doctest: +SKIP
```

Reply:
maybe we don't want to skip it but we can match the output with an ellipsis.

Reply:
Am I doing this correctly in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/ef925fd148e68154b0cae7fa5f209f176e26fb71?

Reply:
Found it. Done in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/c614425c060ae09ab05db2b47ef40e8dab5401fb